app/presenters/publishing_api_presenters.rb

```diff
@@ -27,6 +18,12 @@ def self.presenter_class_for(model)
       PublishingApiPresenters::TakePart
     when Topic
       PublishingApiPresenters::PolicyAreaPlaceholder
+    when ::StatisticsAnnouncement
+      if model.unpublished? || (model.publication && model.publication.published?)
```

Comment:
I feel like this logic should be wrapped up behind a single predicate in `StatisticsAnnouncement`.